tests/utils_tests/files/strip_tags2.txt

```diff
@@ -93,15 +93,15 @@ If the `minSdk` is API 11 or higher, then we can leave our inheritance to `Activ
 ##### a. Static approach to hosting the Fragments
 ###### Adding `` element in activity layout
 
-> [activity_static.xml](https://github.com/iontech/Fragments_Example/blob/master/res/layout/activity_static.xml)
+> [activity_static.xml](https://github.com/iontech/Fragments_Example/blob/main/res/layout/activity_static.xml)
 
 Inorder to statically add a Fragment into your Activity, just add `` element with the necessary layout attributes and the `android:name` attribute set to the fully qualified class name of the corresponding Fragment.
 
 
 ##### b. Dynamic approach
 ###### Using `FragmentTransaction`s
 
-> [ADynamicFragmentActivity.java](https://github.com/iontech/Fragments_Example/blob/master/src/main/java/com/github/iontech/fragments_example/ADynamicFragmentActivity.java)
+> [ADynamicFragmentActivity.java](https://github.com/iontech/Fragments_Example/blob/main/src/main/java/com/github/iontech/fragments_example/ADynamicFragmentActivity.java)
```

Comment:
Can I suggest maybe to change these urls to be example.com something? It's a bit weird for djangos code to point to some random users repo, a repo that also doesn't exist.

Reply:
I agree - weird to even include this text.

I've rewritten the whole file as "Lorem ipsum" retaining various markdown formatting, and inline HTML, the main point of the test.